spec/controllers/users_controller_spec.rb

```diff
@@ -27,18 +27,32 @@
     let(:user_params) { Fabricate.attributes_for(:user) }
 
     it_behaves_like 'ApplicationController#require_logout' do
```

Comment:
You would generally name a shared example as a behavior or scenario. `it_behaves_like "only logged out users can access"`. Something like that.